browser-tests/utils/sharedTests.ts

```diff
@@ -0,0 +1,175 @@
+import shared from '../selectors/shared';
```

Comment:
should we move this file to `browser-tests/sharedTests` it's not really an util?